### PR TITLE
Minor English grammar fixes

### PR DESCRIPTION
**Pull request by spook**
I made some minor English grammar corrections.

The note about `ebtables` confused me.  Please verify that my re-write is correct before merging.  I may have it backwards!

### CODE REVIEW

**Comment by ilario** [comment]
Thanks! All the changes look great!
Regarding the ebtables note: I would replace your "bridge prevents the layer 2 packets, which are sent to the anycast address, from propogating through the cloud" with "bridge prevents packets sent to the anycast address from propogating through the cloud", hope the meaning is more clear now :)

File: howitworks.txt [hunk 1]
```diff
@@ -13,22 +13,22 @@ icon:edit[] link:https://github.com/libremesh/lime-web/edit/master/howitworks.tx
...
 The cloud layer 2 uses the dynamic routing protocol BATMAN-ADV
 
 image::img/bat-adv.png[batmanadv,100,float="left"]
 *******************
-B.A.T.M.A.N. advanced is a mesh routing protocol which runs in kernel space.
-Even if the network topology is made by multiple nodes and multiple hops, abstracts it to a single layer 2 collision domain.
-So from the user perspective the entire mesh will look like a single LAN.
-This architecture is very interesting for roaming purposes, thus TCP and UDP connections are not lost even when moving and changing AP.
+B.A.T.M.A.N. Advanced is a mesh routing protocol that runs in kernel space.
+Even if the network topology is made of multiple nodes and multiple hops, B.A.T.M.A.N. Advanced abstracts it to a single layer 2 collision domain.
```
Comment (G10h4ck) [inline howitworks.txt]: In this sentence 'collision domain' must be changed to 'broadcast domain'

Comment (spook) [reply howitworks.txt]: Done, thanx.

**Comment by spook** [comment]
@ilario - thanx for the clarification on `ebtables`.  Your phrasing is better, I changed it to that.

**Comment by ilario** [comment]
Thanks for the contribution! Merging :)

**Pull request merged**